OJS.Workers.ExecutionStrategies/CSharp/DotNetCore/DotNetCoreCompileExecuteAndCheckExecutionStrategy.cs

```diff
@@ -149,5 +167,32 @@ private void CreateRuntimeConfigJsonFile(string directory, string text)
 
             File.WriteAllText(jsonFilePath, text);
         }
+
+        private void AddUsingNamespacesForDotNetSix<T>(IExecutionContext<T> executionContext)
+        {
+            if (this.Type != ExecutionStrategyType.DotNetCore6CompileExecuteAndCheck)
+            {
+                return;
+            }
+
+            var stringBuilder = new StringBuilder();
+
+            this.DotNetSixDefaultUsingNamespaces
+                .ForEach(usingNamespace => this.AddUsingNamespaceIfNotExist(usingNamespace, stringBuilder, executionContext.Code));
+
+            stringBuilder.AppendLine(executionContext.Code);
+
+            executionContext.Code = stringBuilder.ToString();
+        }
+
+        private void AddUsingNamespaceIfNotExist(string usingNamespace, StringBuilder sb, string code)
+        {
+            if (code.Contains(usingNamespace))
+            {
+                return;
+            }
+
+            sb.AppendLine(usingNamespace);
+        }
     }
```

Comment:
`StringBuilder` is nice touch, but it is mutable structure. Here it is not that important do to this, so we can just make a method:
```
GetNamespaceIfNotExist(string usingNamespace, string code) {
  return code.Contains(usingNamespace)
      ? string.Empty
      : usingNamespace;
}
```